README.md

```diff
@@ -231,6 +231,30 @@ console.log('index name =', indexName)
 
 For more detailed information, see the [indexing strategies page](https://docs.mongodb.com/manual/applications/indexes/).
 
+## Error Handling
+
+If you need to filter certain errors from our driver we have a helpful tree of errors described in [docs/errors.md](https://github.com/mongodb/node-mongodb-native/blob/4.1/docs/errors.md).
+We expect `instanceof` checks to be maintained as our error API under semver as described in the hierarchy errors.md document.
```

Comment:
The "hierarchy errors.md document" bit reads a little strange to me, could we improve the phrasing there? Also "as our error API" -> "in our error API" ?